### ISSUE TITLE
Lab FE QA Notes

### ISSUE THREAD

**Issue by samWinfield** [bluecadet/hscrb-website#127]
Lab Nav: 

- [ ] Make the lab nav the same height as the sticky nav, and modify the position of the sticky nav (only on the lab page) so the two elements are aligned.
- [ ] Add box-shadow{0px 4px 8px #0001} to the lab nav
- [ ] Remove transparency from lab-nav element.

![image](https://user-images.githubusercontent.com/19784290/56616005-715a3280-65ea-11e9-904b-65b7fcea160c.png)

Publications:

Not sure the right way to implement, but because the background here is receiving a 'background-color' of #fbfbfb, it's blending in with the background of the publication cards. Can we remove the background here? Does that make things weird elsewhere? 

I think we should also add a 3.2rem margin above each year in the list.

![image](https://user-images.githubusercontent.com/19784290/56616460-7966a200-65eb-11e9-8bbe-b36a64edf910.png)

**Comment by maxfenton** [comment]
@samWinfield 
> Can we remove the background here? Does that make things weird elsewhere?

Which background? The grey on the page behind the list?

**Comment by samWinfield** [comment]
@maxfenton Yes, sorry. I was referring to the grey on the page behind the list.

**Comment by maxfenton** [comment]
@samWinfield Let's do a 20-min screenshare together on this one to try some things out with the lab nav height, etc. (Please)
